src/v3/src/components/PasswordRequirements/PasswordRequirementListItem.tsx

```diff
@@ -32,9 +32,7 @@ const PasswordRequirementListItem: FunctionComponent<PasswordRequirementProps> =
         display="flex"
         alignItems="center"
       >
-        <Box sx={{ minInlineSize: (theme) => theme.spacing(2) }}>
-          <Icon status={status} />
-        </Box>
+        <Icon status={status} />
```

Comment:
Can we add a comment or maybe some kind of test (not sure how) to ensure we don't regress? Especially since we're probably going to need to change this code again in the Odyssey 1.x upgrade.

Reply:
Given the issue was the size of the image, it can only be tested using a VRT. I will talk to Vijet/Venkata to see if there is a list of things that we want to target with VRT and get this added to the list. I will also add a comment here as well.